lib/browser.js

```diff
@@ -18,3 +18,4 @@ require('./http/xhr');
 
 if (typeof window !== 'undefined') window.AWS = AWS;
 if (typeof module !== 'undefined') module.exports = AWS;
+if (typeof self !== 'undefined') module.exports = AWS;
```

Comment:
@cesarpachon 
Did you mean to set `self.AWS = AWS;` instead of `module.exports = AWS;`?